app/store/selectors/currentAthlete.js

```diff
@@ -0,0 +1,2 @@
+export default state => state.entities.athletes.currentAthlete;
```

Comment:
this make more sense 
` state.entities.athletes[currentAthlete]`
even if it is the same